components/config-mgmt-service/backend/elastic/runs.go

```diff
@@ -89,12 +89,12 @@ func (es Backend) GetDeletedCountsTimeSeries(
 	copiedFilters["exists"] = []string{"false"}
 	mainQuery := newBoolQueryFromFilters(copiedFilters)
 
-	deletedNodePeroids := make([]backend.CountPeroid, getNumberOf24hBetween(startTime, endTime))
+	deletedNodePeriods := make([]backend.CountPeroid, getNumberOf24hBetween(startTime, endTime))
```

Comment:
Updated the variable using VSCode `Rename Symbol(F2)` so that it can takes care of references.